documents/minutes/2018-12-02 WCA Board Meeting.md

```diff
@@ -0,0 +1,32 @@
+<div class="text-center">
+MINUTES OF MEETING OF THE BOARD OF DIRECTORS OF WORLD CUBE ASSOCIATION
+
+A California Nonprofit Public Benefit Corporation
+</div>
+
+On December 2, 2018 at 9:13 a.m. PST, the Board of Directors of World Cube Association (the “Board”) met for a regular meeting via Google Hangouts. The following directors were present at the meeting:
+
+<div class="text-center">
+Alberto Pérez de Rada Fiol <br>
+Christopher Wright <br>
+Olivér Perge <br>
+Robert Burton <br>
+</div>
```

Comment:
I would use an empty line before the names and the title.

Reply:
There's one! (Line 15)